### ISSUE TITLE
release upload error occurred on travis

### ISSUE THREAD

**Issue by shimizukawa** [jazzband/django-redshift-backend#58]
by #55, we have upload script by using jazzband bot that invoked from travis.
However we can't upload release files by error.

Travis log: https://travis-ci.org/jazzband/django-redshift-backend/jobs/485747325#L390

> HTTPError: 400 Client Error: File already exists. for url: https://jazzband.co/projects/django-redshift-backend/upload

refs:
* https://github.com/jazzband/django-redshift-backend/issues/51#issuecomment-458469824

@jezdez Can you check this issue?

**Comment by shimizukawa** [comment]
Sorry, I overlooked the mail from jazzband that contains release procedure.
I got a 1.0.0a2 release on PyPI!

**Comment by jezdez** [comment]
Ah interesting, @shimizukawa, I'm glad it worked out.

Don't forget that there are multiple files uploaded to Jazzband.co (the whl file and the tar.gz file). I've yet to add a feature to release all files of a version at once 😬 and you'll need to release the files individually right now.

Also, seeing the deploy failure in Travis, have you by any chance restarted the Travis build process for the tag commit?

**Comment by jezdez** [comment]
@shimizukawa Ah nevermind, I saw in https://github.com/jazzband/django-redshift-backend/issues/51#issuecomment-458469824 that you've rebuilt the tag on Travis. It just overwrote the build history and doesn't show the successful deploy to Jazzband.co anymore. Apologies if the process is somewhat confusing still. 🙇

**Comment by shimizukawa** [comment]
> you've rebuilt the tag on Travis

Exactly. I did not understand well, just rebuilt thinking that travis did not respond to tags.

I have considered the reason why I did not notice the mail.
It is written on the page https://jazzband.co/about/releases as follows.
```
They will be able to confirm the uploads and release them to PyPI individually.
```
However, I was not able to understand that "confirm" in this sentence was "When mail notification arrives from jazzband, I will confirm it".

I hope this information will be useful. Thanks!
